pkg/controller/master/image/register.go

```diff
@@ -11,6 +11,7 @@ import (
 const (
 	vmImageControllerName      = "vm-image-controller"
 	backingImageControllerName = "backing-image-controller"
+	cronJobControllerName      = "cron-job-controller"
```

Comment:
This variable is unused.

Reply:
This variable is used here https://github.com/harvester/harvester/blob/a128dbfcdc2df4329ccd342ed4ae54f44e4cb432/pkg/controller/master/schedulevmbackup/register.go#L68

Reply:
There are two `cronJobControllerName` variables. One is in image and another is in schedulevmbackup. I use IDE reference check and can't find usage for the one in image package. Could you check it again? Thanks.

Reply:
Sorry, it's my fault, you're right, thanks for pointing this out. I will update it, thanks.

Reply:
Updated per suggestion, thanks.